Cannot build library

**Describe the bug**

I cannot build Hashcat library with the newest code.

**To Reproduce**

1. Download the newest code.
2. Run `make libhashcat.so.6.2.5 hashcat`.
3. See error.

```
/usr/bin/ld: obj/emu_inc_bignum_operations.NATIVE.o: in function `mul_mod':
emu_inc_bignum_operations.c:(.text+0x36c0): multiple definition of `mul_mod'; obj/emu_inc_ecc_secp256k1.NATIVE.o:emu_inc_ecc_secp256k1.c:(.text+0x16b0): first defined here
collect2: error: ld returned 1 exit status
make: *** [src/Makefile:644: libhashcat.so.6.2.5] Error 1
make: *** Waiting for unfinished jobs....
```

**Expected behavior**

Library should be built successfully.

**Hashcat version:**

 - OS: Linux
 - Distribution: Ubuntu 20.04
 - Version: v6.2.5+ (commit 372d3a127)

**Additional context**

Probably problem occurred after merging PR GH-3295 (my comment https://github.com/hashcat/hashcat/pull/3295#issuecomment-1161384563).

--- a/OpenCL/inc_bignum_operations.cl
+++ b/OpenCL/inc_bignum_operations.cl
@@ -846,7 +846,7 @@ DECLSPEC void mul_masked (PRIVATE_AS u32 *r, PRIVATE_AS const u32 *x, PRIVATE_AS
 // m   [128], modulo
 // fact[ 64], our m' (actually it is fact[65])
 
-DECLSPEC void mul_mod (PRIVATE_AS u32 *x, PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact)
+DECLSPEC void mul_mod128 (PRIVATE_AS u32 *x, PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact)
 {
   // 1st multiplication
   // p = x * y
@@ -1411,7 +1411,7 @@ DECLSPEC void pow_mod_precomp_g (PRIVATE_AS u32 *r, PRIVATE_AS const u32 *b_pre,
 
     const u32 bit_set = (y[div] >> mod) & 1;
 
-    if (bit_set == 1) mul_mod (r, b_pre + i * 64, m, fact);
+    if (bit_set == 1) mul_mod128 (r, b_pre + i * 64, m, fact);
   }
 }
 
@@ -1445,9 +1445,9 @@ DECLSPEC void pow_mod (PRIVATE_AS u32 *r, PRIVATE_AS u32 *x, PRIVATE_AS const u3
 
     const u32 bit_set = (y[div] >> mod) & 1;
 
-    if (bit_set == 1) mul_mod (r, x, m, fact);
+    if (bit_set == 1) mul_mod128 (r, x, m, fact);
 
-    mul_mod (x, x, m, fact);
+    mul_mod128 (x, x, m, fact);
   }
 }
 

--- a/OpenCL/inc_bignum_operations.h
+++ b/OpenCL/inc_bignum_operations.h
@@ -9,7 +9,7 @@
 DECLSPEC void mod_4096   (PRIVATE_AS u32 *n, PRIVATE_AS const u32 *m);
 DECLSPEC void mul        (PRIVATE_AS u32 *r, PRIVATE_AS const u32 *x, PRIVATE_AS const u32 *y);
 DECLSPEC void mul_masked (PRIVATE_AS u32 *r, PRIVATE_AS const u32 *x, PRIVATE_AS const u32 *y);
-DECLSPEC void mul_mod    (PRIVATE_AS u32 *x, PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact);
+DECLSPEC void mul_mod128 (PRIVATE_AS u32 *x, PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact);
 
 DECLSPEC void pow_mod_precomp_g (PRIVATE_AS u32 *r, PRIVATE_AS const u32 *b_pre, PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact);
 DECLSPEC void pow_mod           (PRIVATE_AS u32 *r, PRIVATE_AS       u32 *x,     PRIVATE_AS const u32 *y, PRIVATE_AS const u32 *m, PRIVATE_AS const u32 *fact);

--- a/OpenCL/m29200_a0-pure.cl
+++ b/OpenCL/m29200_a0-pure.cl
@@ -276,7 +276,7 @@ KERNEL_FQ void m29200_mxx (KERN_ATTR_RULES_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];
@@ -544,7 +544,7 @@ KERNEL_FQ void m29200_sxx (KERN_ATTR_RULES_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];

--- a/OpenCL/m29200_a1-pure.cl
+++ b/OpenCL/m29200_a1-pure.cl
@@ -279,7 +279,7 @@ KERNEL_FQ void m29200_mxx (KERN_ATTR_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];
@@ -552,7 +552,7 @@ KERNEL_FQ void m29200_sxx (KERN_ATTR_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];

--- a/OpenCL/m29200_a3-pure.cl
+++ b/OpenCL/m29200_a3-pure.cl
@@ -286,7 +286,7 @@ KERNEL_FQ void m29200_mxx (KERN_ATTR_VECTOR_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];
@@ -567,7 +567,7 @@ KERNEL_FQ void m29200_sxx (KERN_ATTR_VECTOR_ESALT (radmin3_t))
         esalt_bufs[DIGESTS_OFFSET_HOST].pre[pre_idx + 63],
       };
 
-      mul_mod (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
+      mul_mod128 (r_t, pre, m, fact); // r_t = (r_t * RADMIN3_PRE[n]) % m
     }
 
     const u32 r0 = r_t[0];

